Packs/GenericSQL/pack_metadata.json

```
@@ -3,7 +3,7 @@
     "description": "Connect and execute sql queries in 4 Databases: MySQL, PostgreSQL, Microsoft SQL Server and Oracle",
     "support": "xsoar",
     "serverMinVersion": "5.0.0",
-    "currentVersion": "1.2.4",
+    "currentVersion": "1.2.5",
```

worth a minor bump, please rename `1_2_5.md` to `1_3_0.md` accordingly

```suggestion
    "currentVersion": "1.3.0",
```